### PR TITLE
PMM-8575 Migrated toolkit to modules

### PR DESCRIPTION
**Pull request by percona-csalguero**
This is draft because before really merging this, I would like to create a new release for Percona Toolkit so we can use semantic versioning and tags.

### CODE REVIEW

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1998-a45287c
client docker - perconalab/pmm-client-fb:PR-1998-a45287c
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1998-a45287c.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-1998-a45287c&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1998-a45287c.tar.gz

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/7851/

**Comment by JNKPercona** [comment]
UI tests have failed, Please check: UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/9817/

File: build/bin/build-client-binary [hunk 1]
```diff
@@ -35,7 +35,8 @@ gobuild_component() {
          [ ${component} = 'postgres_exporter' ]; then
         build_command="cd /tmp/go/src/${extract_path} && go build -o \${binary_dir}/${result_file} ./${component_path}"
     else
-        build_command="go build -o \${binary_dir}/${result_file} ./src/${extract_path}/${component_path}"
+        build_command="cd /tmp/go/go build -o \${binary_dir}/${result_file} ./src/${extract_path}/${component_path}"
+        build_command="cd /tmp/go//src/${extract_path}/${component_path} && go build -o \${binary_dir}/${result_file} ."
```
Comment (nikita-b) [inline build/bin/build-client-binary:39]: `/tmp/go//src/` two slashes

**Review by nikita-b — APPROVED** [review]

**Comment by percona-csalguero** [comment]
Closing because to modify the requested files, I need to recreate the branch due to git merge conflicts in submodules

**Pull request closed without merging**